modules/ROOT/pages/release-notes.adoc

```diff
@@ -10,6 +10,16 @@ and either Couchbase Analytics Service (CBAS) or tabular views of your Capella c
 
 For self-managed CBAS, the Power BI connector requires Couchbase Server 7.2.4 or newer.
 
+[#v-1-1]
+== Version 1.1 (2024-06-12)
+
+https://packages.couchbase.com/releases/couchbase-powerbi-connector/1.0/couchbase-powerbi-connector-1.0.mez[Power BI Connector Download]
+
+The supported and tested dependencies of this release are:
+
+* https://packages.couchbase.com/releases/couchbase-odbc-driver/1.1/couchbase-odbc-1.1-win64.msi[couchbase-odbc-1.1-win64.msi]
+
+** See the https://packages.couchbase.com/releases/couchbase-odbc-driver/1.1/couchbase-odbc-driver-1.1-notices.txt[Notices File] for dependency licenses for the ODBC Driver.
 
 [#v-1-0]
 == Version 1.0 (2024-01-25)
```

Comment:
If we keep version 1.0, should we specify that it only works with OpenSSL 1.1.1?

Reply:
Yes, I should probably note OpenSSL dependencies here.